.cirrus.yml

```diff
@@ -92,7 +92,11 @@ gce_instance:
     # the git-validate tool which are difficult to debug and fix.
     skip: $CIRRUS_PR == ''
 
-    timeout_in: 30m
+    timeout_in: 10m
```

Comment:
Just curious why you lowered the time?

Reply:
To avoid leaving a big-expensive VM hanging around any longer than necessary.  For this task, if it takes longer than a minute or three, it's likely somethings very wrong and it needs to timeout.

Reply:
Do you think this is worthy of a comment in the code?

Reply:
Wouldn't hurt, but I'd add it later.  I'd like to get this in sooner, especially given the green test buttons.